libs/auth/testing/src/inmemory-backend/auth.service.integration.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClient, HttpClientModule } from '@angular/common/http';
+import { HttpClientInMemoryWebApiModule, STATUS } from 'angular-in-memory-web-api';
+
+import { DaffInMemoryBackendAuthService } from './auth.service';
+
+describe('DaffAuthInMemoryBackend | Integration', () => {
+  let httpClient;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        HttpClientModule,
+        HttpClientInMemoryWebApiModule.forRoot(DaffInMemoryBackendAuthService)
+      ],
+    });
+
+    httpClient = TestBed.get(HttpClient);
+  });
+
+  describe('processing a login request', () => {
+    it('should process post requests of the form `/api/auth/login` and return a token', done => {
+      httpClient.post('/api/auth/login', {}).subscribe(result => {
+        expect(result).toBeDefined();
```

Comment:
Is being `defined` sufficient evidence that this test passes?

Reply:
I can check that its token is a string. Without mocking internals, I can't know the value of the token.

Reply:
I checked for a defined token field on the result. Still a stronger check than I had.